### ISSUE TITLE
Dire and radiant base are very different (ancient true sight is different)

### ISSUE THREAD

**Issue by Yan00r** [jeffhill/Dota2#703]
### Description

Dire and radiant base are very different, and in equal situations, this gives advantages to opponents besieging the base of dire, but does not give that advantage to opponents besieging the base of radiant. In that case dire side must buy sentry ward to see enemy who walk around in invisibility, when radiant don't need this ward because invisible hero can't across form one side of base to another without been seen.

### Screenshots

Radiant true sight from ancient
![image](https://user-images.githubusercontent.com/106772815/172503809-5bc64c1f-6373-49d9-8c0a-0de87f5b6167.png)
Dire true sight from ancient 
(There can bee seen small way without true sight [in the middle of ancient an fountain] where enemy heroes can move free if they have invisibility)
![image](https://user-images.githubusercontent.com/106772815/172505400-992be1e0-8853-4f44-859a-bd94d9b8f5ed.png)

**Comment by Avarise831** [comment]
tier 3-4 range towers same shit

**Comment by Yan00r** [comment]
yes, at night in radiant side you can't walk witout seen
![image](https://user-images.githubusercontent.com/106772815/172507610-ef88f860-62c0-4140-ac46-d69f2a6503cc.png)
But you can do this at dire side, and there no need any invisibility to be not seen by anybody
![image](https://user-images.githubusercontent.com/106772815/172507715-3b594180-79c3-4926-8267-b2a53949cf43.png)

**Comment by Avarise831** [comment]
> yes, at night in radiant side you can't walk witout seen ![image](https://user-images.githubusercontent.com/106772815/172507610-ef88f860-62c0-4140-ac46-d69f2a6503cc.png) But you can do this at dire side, and there no need any invisibility to be not seen by anybody ![image](https://user-images.githubusercontent.com/106772815/172507715-3b594180-79c3-4926-8267-b2a53949cf43.png)

yes but i mean u know when u play furion and try to kill couriers at start game  between the towers its weird XD

**Comment by Avarise831** [comment]
https://www.reddit.com/r/DotA2/comments/ulpopu/dota_2_bug_tier_34_range_towers_what_is_that_how/

**Comment by Yan00r** [comment]
i just forget for this small way without damage of one of two towers, yes big problem
